lib/utils.js

```diff
@@ -88,11 +88,38 @@ function enable2FA(action_alias) {
          action_alias.extra.security.twofactor !== undefined;
 }
 
+function normalizeAddressee(msg, adapter) {
```

Comment:
I really wish we had a better code organization - ideally have a module per adapter, etc.

Anyway, that's fine for now but something we can hopefully improve down the road :)